src/core/src/main/resources/org/apache/jmeter/resources/messages_zh_CN.properties

```diff
@@ -427,6 +428,8 @@ headers_stored=信息头存储在信息头管理器中
 heap_dump=创建一个堆转储
 help=帮助
 help_node=这个节点是什么？
+hide JMeter=隐藏 JMeter
```

Comment:
There is no such key in JMeter (at least I didn't find it). Same for the next key `hide Others`.